app/weather_parser.py

```diff
@@ -0,0 +1,46 @@
+import forecastio
+import json
+import requests
+from config import FORECAST_API_KEY, GOOGLE_API_KEY
+
+
+forecast_api_key = FORECAST_API_KEY
+geocode_api_key = GOOGLE_API_KEY
+geocode_url = 'https://maps.googleapis.com/maps/api/geocode/json?'
+place_type = None
+formatted_address = None
+
+
+class WeatherForecast:
+
+    def __init__(self):
+        self.place_type = None
+        self.formatted_address = None
+
+    def get_requests(self, address):
+        geocodes = self.get_geocode_request(address=address)
+        place_query_status = geocodes['status']
+        if place_query_status == 'OK':
+            self.place_type = geocodes['results'][0]['address_components'][0]['types'][0]
+            self.formatted_address = geocodes['results'][0]['formatted_address']
+            latitude = geocodes['results'][0]['geometry']['location']['lat']
+            longitude = geocodes['results'][0]['geometry']['location']['lng']
+            return self.get_forecast_request(latitude=latitude, longitude=longitude)
+        return str(place_query_status)
+
+    def generate_forecast(self, address, time_method):
+        forecast = self.get_requests(address)
+        if time_method == 'daily':
+            return forecast.daily()
+        if time_method == 'hourly':
```

Comment:
yeah, these ifs indicate a too large PR. should have seen it from your title: “app can generate additional daily and hourly forecast” — there is an “and”, which usually means “two things” -> two PRs